core/nlp/README.md

```diff
@@ -2,23 +2,92 @@
 
 ## Description
 
-The lib provides you tools for nlp
+The lib provides the following tools for NLP:
+- `spacy_tokenizer.py`
+    - Tokenization, Lemmatization & other components available in spaCy
+    - By default the requirements.txt includes models for several common languages that can be used by calling `MultilingualTokenizer(use_models=True,)`. These tend to have better performance than the default rule-based system, but if not needed feel free to remove them from the requirments.
+- `language_detection.py`
+    - Language detection
+- `text_cleaner.py`
+    - Text cleaning (e.g. remove emojis)
+- `symspell_checker.py`
+    - Spell checking
 
 ## Examples
 
-Here is an example of usage
+Clone & cd into the repository and install the `core/nlp/requirements.txt` packages.
```

Comment:
No need to say "clone & cd" - it is implied by the fact this lives on github

Reply:
Hahah I always think back the first time I tried to figure out how to download a repository from github and ended up downloading the zip files cuz I had no clue what clone means 😂
Committed👍